code/modules/spells/spell_types/ethereal_jaunt.dm

```diff
@@ -30,6 +30,7 @@
 			animation.layer = 5
 			animation.master = holder
 			target.ExtinguishMob()
+			mutations = list(MUT_MUTE)

```

Comment:
```wiz.mutations |= MUT_MUTE```

will add mute if it's not present, and will do nothing if it is present.
does not override other mutations.